src/components/shared/Modal.css

```diff
@@ -1,19 +1,23 @@
 .modal-wrapper {
   position: fixed;
   display: flex;
+  flex-direction: column;
   align-items: center;
+  justify-content: flex-start;
```

Comment:
we don't need this property since it is the default. making it explicit as it is here would suggest that it is overridden somewhere else.